docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3.9"
 x-posgis_environment: &postgis_environment
   POSTGRES_DB: ${POSTGRES_DB:-postgres}
   POSTGRES_USER: ${POSTGRES_USER:-postgres}
-  POSTGRES_HOST: ${POSTGRES_HOST:-gis_postgis}
```

Comment:
POSTGRES_HOST isn't really configurable from the loaders perspective, due to this docker file, the loader will always see the host as `gis_postgis`